lib/cdo/lesson_import_helper.rb

```diff
@@ -108,6 +111,20 @@ def self.create_lesson_vocabularies(cb_vocab, course_version_id)
     end
   end
 
+  def self.find_lesson_programming_expressions(cb_blocks)
+    return [] if cb_blocks.blank?
+    cb_blocks.map do |cb_block|
+      raise unless cb_block['slug'] && cb_block['parent_slug']
+      programming_environment = ProgrammingEnvironment.find_by(name: cb_block['parent_slug'])
+      block = ProgrammingExpression.find_by(
+        programming_environment_id: programming_environment.id,
+        name: cb_block['slug']
+      )
+      log "Could not find #{cb_block['slug']} for #{cb_block['parent_slug']}" unless block
```

Comment:
are we hitting this log message at all?

Reply:
Now that you ask I don't think so. I think I just got a different kind of error when it could not find the block. I fixed all the cases where that was the case as I was testing.